src/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders Standalone Editor without config', () => {
```

Comment:
That's a good point, we better just convert the test for the base64 file instead of adding this test to the PR.
I updated the PR.
About files organization, I see that also Create React App prefers to keep tests close to the relative implementation files. See for example at their [website](https://reactjs.org/docs/faq-structure.html) or in their [Getting started guide](https://jestjs.io/docs/en/getting-started).
The rationale is to keep tests (especially unit) easy to find next to source files. There are older test frameworks that put all the tests in a separate directory; while Jest does support it, it might not be as easy to keep tests organized and discoverable. But that's debatable of course

Reply:
Right now we don't have many sub-directories so i kind of fear having too many files in the same directory with relatively close names.
I'll go have a look on what other projects do